docs/writing-docs/doc-blocks.md

```diff
@@ -0,0 +1,233 @@
+
+import { Meta, Primary, Controls, Story } from '@storybook/blocks';
+import * as ButtonStories from './Button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+A button is ...
+
+<Primary />
+
+## Props
+
+<Controls />
+
+## Stories
+
+### Primary
+
+Button can be of primary importance
+
+<Story of={ButtonStories.Primary} />
+
+Button can be of secondary importance
+
+<Story of={ButtonStories.Secondary} />
+
+{/* ... */}
+```
+<!-- prettier-ignore-end -->
+
+## Customizing the automatic docs page
+
+The blocks are also used to define the page template for [automatics docs](./autodocs.md). For example, here's the default template:
+
+![Screenshot of automatic docs template](TK)
+
+```jsx
+import { Title, Subtitle, Description, Primary, Controls, Stories } from '@storybook/blocks';
+
+export const autoDocsTemplate = () => (
+  <>
+    <Title />
+    <Subtitle />
+    <Description />
+    <Primary />
+    <Controls />
+    <Stories />
+  </>
+);
+```
+
+If you [override the default page template](./autodocs.md#write-a-custom-template), you can use doc blocks in a similar manner to build the perfect docs page for your project.
+
+Note that some doc blocks render other blocks. For example, the `<Stories />` block expands to:
+
+<!-- prettier-ignore-start -->
+```md
+## Stories
+
+<Canvas>
+  ### Story name
+  <Description />
+  <Story />
+  <Source />
+</Canvas>
+
+{/* ... repeat <Canvas> for each story */}
+```
+<!-- prettier-ignore-end -->
+
+As a result, for example, customizing the [`Source`](../api/doc-block-source.md) block via parameters (see next section) will also affect the Source blocks rendered as part of [`Canvas`](../api/doc-block-canvas.md) blocks.
+
+## Customizing doc blocks
+
+In both use cases (MDX and automatic docs), many of the doc blocks can be customized via [parameters](../writing-stories/parameters.md).
+
+For example, you can filter out the `style` prop from all [`Controls`](../api/doc-block-controls.md) tables through your Storybook:
+
+<!-- prettier-ignore-start -->
+```js
+// .storybook/preview.js
+
+export const parameters = {
+  docs: {
+    controls: { exclude: ['style'] }
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+<div class="aside">
+
+ℹ️ Parameters can also be defined at the [component](../writing-stories/parameters.md#component-parameters) (or meta) level or the [story](../writing-stories/parameters.md#story-parameters) level, allowing you to customize doc blocks exactly as you need, where you need.
+
+</div>
+
+The blocks that accept customization via parameters are marked in the list of available blocks, below.
+
+When using a doc block in MDX, it can also be customized with its props:
+
+<!-- prettier-ignore-start -->
+```md
+<Controls exclude={['style']}>
+```
+<!-- prettier-ignore-end -->
+
+## Available blocks
+
+Each block has a dedicated API reference page, detailing usage, available options, and technical details.
+
+If a doc block accepts parameters, it is marked with **(P)**.
```

Comment:
Really not sure about this idea. Very welcome to suggestions.

Reply:
I'm not sure it's needed? I would think it's self-explanatory that some of them reacts to `parameters.docs.X`, some don't.

Eg. `ColorPalette` doesn't because it doesn't reference any stories/components, so there's no where to pull parameters from. I don't think a user would try that.

But then I also see that `Description` don't have a (P), which definitely reads from parameters, so maybe I'm misunderstanding the idea here.

Reply:
`Description` is an odd duck. The content is provided via parameters, but the block isn't _configured_ that way. 🤷‍♂️ 

The thought behind marking the blocks was an attempt at helping people understand (_before_ clicking through to the API reference) which blocks can be configured. But I have another idea...